src/tests/functional/plugin/shared/src/snippets/mlp.cpp

```diff
@@ -0,0 +1,119 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "snippets/mlp.hpp"
+
+#include <common_test_utils/ov_tensor_utils.hpp>
+
+#include "common_test_utils/common_utils.hpp"
+#include "functional_test_utils/skip_tests_config.hpp"
+#include "subgraph_mlp.hpp"
+
+namespace ov {
+namespace test {
+namespace snippets {
+
+void MLPBase::compile_model() {
+    if (m_thread_count != default_thread_count)
+        core->set_property(targetDevice, ov::inference_num_threads(m_thread_count));
+    SubgraphBaseTest::compile_model();
+}
+
+void MLPBase::SetUp() {
+    std::vector<InputShape> input_shapes;
+    ov::element::Type prc;
+    ov::AnyMap additional_config;
+    init_params(input_shapes, prc, additional_config);
+    init_input_shapes(input_shapes);
+
+    const auto subgraph_model = get_subgraph(m_num_hidden_layers, m_hidden_matmul_size);
+    function = subgraph_model->getOriginal();
+
+    configuration.insert(additional_config.begin(), additional_config.end());
+    if (!configuration.count("SNIPPETS_MODE")) {
+        configuration.insert({"SNIPPETS_MODE", "IGNORE_CALLBACK"});
+    }
+
+    inType = outType = prc;
+    setInferenceType(prc);
+    init_thresholds();
```

Comment:
Could you please check if we need `init_thresholds()` call here? It is already called in base class method `SubgraphBaseTest::compare`, and it seems like we don't need to init thresholds before that

If it is not needed, please also remove `init_thresholds()` call in base MHA test class